kie-server-parent/kie-server-controller/kie-server-controller-test-war/src/main/assembly/assembly-ee6-container.xml

```diff
@@ -50,6 +50,7 @@
 
         <include>com.sun.xml.bind:jaxb-impl</include>
         <include>com.sun.xml.bind:jaxb-core</include>
+        <include>org.slf4j:slf4j-jdk14</include>
```

Comment:
Can be, will check it.
It makes me think... this dependency is used also in Kie server ee6 WAR. Is there some special reason why it is used there or slf4j-api should be more appropriate there too?